### ISSUE TITLE
x64 Crashes on Session Logon

### ISSUE THREAD

**Issue by wondernaut** [stephenegriffin/mfcmapi#48]
Session/Logon Immediately crashes the app.  I've tried this on MFCMapi.exe.x64.17.0.17099.01, MFCMapi.exe.x64.17.0.17069.01, and MFCMapi.exe.x64.17.0.17033.02

```
Source
MFCMAPI x64

Summary
Stopped working

Date
‎5/‎4/‎2017 3:41 PM

Status
Report sent

Description
Faulting Application Path:	C:\Apps\MFCMAPI.exe

Problem signature
Problem Event Name:	BEX64
Application Name:	MFCMAPI.exe
Application Version:	17.0.17099.1
Application Timestamp:	58ea5e6c
Fault Module Name:	OLMAPI32.DLL
Fault Module Version:	16.0.7967.6531
Fault Module Timestamp:	58f77e35
Exception Offset:	00000000001cc358
Exception Code:	c0000409
Exception Data:	0000000000000007
OS Version:	10.0.14393.2.0.0.256.48
Locale ID:	1033
Additional Information 1:	3f46
Additional Information 2:	3f4655693f23e39527312bb59ef8e175
Additional Information 3:	36a5
Additional Information 4:	36a5db6f311abacb4ef1cf62bb74b575

Extra information about the problem
Bucket ID:	53490bf6783614a65b6ae16264d98cff (116372456740)
```

I have 1 Outlook profile.  I made 1 change and it stopped crashing: 
`Tools/Options/Force the use of Outlook's MAPI files`
Previous versions of this (that I used before Windows 10 or Outlook 2016 or Office365) I never had to do this. Maybe it's documented somewhere, but I couldn't find it.  As this is a new platform (Win10/Outlook2016/ExOL), I'm not really sure this is a bug so much as just a learning curve and some documentation.
PS> couldn't find any debug logs.

**Comment by stephenegriffin** [comment]
First things first: not MFCMAPI's fault. No, you shouldn't need to change how you load MAPI. This is a bug in MAPI (or, more precisely, a bug in MSO/Click2Run).
Second (this is for me): I mapped this back to this bucket: fail_fast_fatal_app_exit_c0000409_olmapi32.dll!msodelayloadhook
Third: You appear to be on Current Channel (CC). Do you repro if you try FRDC instead?

**Comment by wondernaut** [comment]
Ok, well that answers that.  I was able to get it to work but just wanted to run it by you in case it was bug.
Sorry, I don't understand FRDC -- are these branches, or releases, or ...? Assuming it's a dev channel, I haven't been able to find it here or in codeplex.

**Comment by stephenegriffin** [comment]
FRDC is a release channel for Office: https://support.office.com/en-us/article/Version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7?ui=en-US&rs=en-US&ad=US&fromAR=1

**Comment by wondernaut** [comment]
Understood!  
Well, I ended up wiping my laptop yesterday for other reasons (possibly it was also creating the problem with the MSO/Click2Run) but loaded CC 1703 Build 7967.2161 on it today and had zero issues with your MFCMapi.exe.x64.17.0.17099.01 build this time.  My guess is if I had loaded a delayed release I would have had the same problem due to a bigger issue with the Win10 install.  Sorry to waste your time.
